internal/gcsx/random_reader.go

```diff
@@ -140,6 +146,19 @@ type randomReader struct {
 	totalReadBytes uint64
 
 	sequentialReadSizeMb int32
+
```

Comment:
can we write a comment above start, limit properties that these properties/variables are used only in case of GCS reads and not for reads from cache.